lib/rules/media-feature-name-no-unknown/index.js

```diff
@@ -1,18 +1,24 @@
 'use strict';
 
+const {
+	isMediaFeature,
+	isMediaQueryInvalid,
+	isGeneralEnclosed,
+} = require('@csstools/media-query-list-parser');
+
 const atRuleParamIndex = require('../../utils/atRuleParamIndex');
 const isCustomMediaQuery = require('../../utils/isCustomMediaQuery');
-const isRangeContextMediaFeature = require('../../utils/isRangeContextMediaFeature');
 const isStandardSyntaxMediaFeatureName = require('../../utils/isStandardSyntaxMediaFeatureName');
 const { mediaFeatureNames } = require('../../reference/mediaFeatures');
-const mediaParser = require('postcss-media-query-parser').default;
 const optionsMatches = require('../../utils/optionsMatches');
-const rangeContextNodeParser = require('../rangeContextNodeParser');
+const parseMediaQuery = require('../../utils/parseMediaQuery');
 const report = require('../../utils/report');
 const ruleMessages = require('../../utils/ruleMessages');
 const validateOptions = require('../../utils/validateOptions');
 const vendor = require('../../utils/vendor');
 const { isRegExp, isString } = require('../../utils/validateTypes');
+const { TokenType, isToken } = require('@csstools/css-tokenizer');
+const { isTokenNode } = require('@csstools/css-parser-algorithms');
```

Comment:
[suggest] Personally, grouping similar imports makes code more readable like this:

```js
const { TokenType, isToken } = require('@csstools/css-tokenizer');
const { isTokenNode } = require('@csstools/css-parser-algorithms');
const {
	isMediaFeature,
	isMediaQueryInvalid,
	isGeneralEnclosed,
} = require('@csstools/media-query-list-parser');
```

What do you think?

Reply:
I agree, thank you for catching these.

_I wished the auto import feature in editors would group these correctly :)_